crates/wysiwyg/src/tests/test_set_content.rs

```diff
@@ -74,3 +74,19 @@ fn set_contents_with_line_break_in_code_block() {
     let model = cm("<pre>\n<code>|Test</code></pre>");
     assert_eq!(tx(&model), "<pre><code>|Test</code></pre>");
 }
+
+#[test]
+fn set_content_from_markdown_blockquote() {
+    let mut model = cm("|");
+    model.set_content_from_markdown(&utf16("> quote")).unwrap();
+    assert_eq!(tx(&model), "<blockquote><p>quote|</p></blockquote>");
+}
+
+#[test]
+fn set_content_from_markdown_with_newlines() {
+    let mut model = cm("|");
+    model
+        .set_content_from_markdown(&utf16("hello\nthere"))
+        .unwrap();
+    assert_eq!(tx(&model), "hello<br />there|");
```

Comment:
Shouldn't this be `<p>hello</p><p>there</p>` ? AFAIK `br` is not well supported yet for all operations of the RTE, e.g. lists, code blocks, etc

Reply:
Yes I think it should. I'll take another look on Monday because I've been working on the element web side of this which is also related.

Reply:
I have removed this check because fixing it is not a small job. I'm going to attack that later this week in a larger piece of markdown conversion work if I get the time